Start with docs. (#44)
FAIRmat-NFDI/pynxtools-spm: docs/how-to-guides/how-to-use-the-reader.md

# How to Use the Reader

## __Input Files__

> The reader requires specific input files for its operation. These files are typically generated by the experiment instrument and include an ELN file with user-provided metadata or an ELN schema file to write your metadata via NOMAD. Below, we discuss the input files and how they are used in different situations.

Reviewer: ```suggestion
The reader requires specific input files for its operation. These files are typically either generated by the experiment instrument or ELN files with user-provided metadata. These ELNS files may also be created from an ELN in NOMAD. Below, we discuss the input files and how they are used in different situations.
```

Reviewer 2: #60

In the tabbed window below, an example ELN schema file for an STS experiment is shown. The schema is designed according to the NeXus application definition `NXsts`. The schema file contains all required metadata fields that are not supplied by the raw file but are required by the `NXsts` application definition. Tabs from left to right show the schema file, the generated archive file, the graphical representation of the archive file in NOMAD, and the final ELN file (with the extension `.yaml`).

=== "STS Schema for NOMAD (sts.scheme.archive.yaml)"
    <div class=scrollable>
    ```yaml

>     definitions:

Reviewer 2: #60

    identifier_experiment: Au_mica_2023_Y_A_diPAMY_154-211C_370C_1min_385C_30min_400C_1min_400C_30min_415_30min_430_30min_11min_30min_30min_20230416_20230420
    user:
    affiliation:
        - Rubel Mozumder
        - Dr. Cojal González, José David
        - Dr. Carlos-Andres Palma
    email:
        - ycjin@physik.hu-berlin.de
        - rubel.mozumder@physik.hu-berlin.de
        - cojal@physik.hu-berlin.de
        - palma@physik.hu-berlin.de
        name: Yichen Jin
    ```
    </div>

### __ELN YAML File__

An ELN YAML file is similar to the YAML file (with the extension `.yaml`) generated by NOMAD from the ELN schema file (with the extension `.scheme.archive.yaml`). This ELN YAML file also contains the same structure as the application definition but is much simpler than the __ELN Schema File__. Such a file can be written by the user following the application definition `NXsts` without the help of NOMAD. This ELN file is only compatible for use with the reader invoked via the command-line interface (see [Transfer Raw Data](../reference/standalone-usages.md) guide).

=== "Eln yaml file (user provided)"
    <div class="scrollable">

>     ```yaml

Reviewer 2: #60
